### PR TITLE
MM-831

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

**Comment by ralder** [comment]
periodically posts list scrolled down  automatically, apparently when receiving event CHANGE_EVENT_STATUSES

File: web/react/components/post_list.jsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ module.exports = React.createClass({
     oldScrollHeight: 0,
     oldZoom: 0,
     scrolledToNew: false,
+    preForcePosision: 0,
```
Comment (jwilander) [inline web/react/components/post_list.jsx]: Position spelt wrong

Comment (nickago) [inline web/react/components/post_list.jsx]: Well that's kinda sad, fix incoming :)

**Comment by nickago** [comment]
Hey @ralder good catch on finding that, it's something we've been trying to fix in this PR for a while. I can't seem to repro in my tests however, were there any specific situations in conjunction with the firing of that event in which this bug became apparent?

**Comment by ralder** [comment]
to reproduce i found two situations:
* 1) open the channel
2) scroll up with mouse
3) change tab in browser to another site, then return and boom! :)

* 1) open the channel
2) scroll up with mouse
3) another member in team click by any place in any channel than second... and boom! :)

windows 8, google chrome stable (ie 11 same)

**Comment by nickago** [comment]
Ready to be re-reviewed

**Comment by ralder** [comment]
almost work (lag only in ie when many messages and read in middle of  posts list, it can be seen how fast scroll down and back) 

may i know why in post_list you didn't same as post_right, i.e. not execute all code in componentDidUpdate if flag is set?

**Comment by jwilander** [comment]
Thanks @ralder for your help on this. @nickago I'm going to see if I can mess with it a bit to get it fixed on IE as well and I'll probably merge sometime this weekend

**Comment by jwilander** [comment]
Submitted a new PR for this

**Pull request closed without merging**